gryphon/lib/exchange/binance_je.py

```diff
@@ -0,0 +1,188 @@
+    }
+
+    def __init__(self, session=None, configuration=None):
+        if self.currencies is None:
+            raise NotImplementedError(
+                "Cannot instantiate BinanceJeExchange. Use a subclass instead."
+            )
+
+        super(BinanceJeExchange, self).__init__(session)
+        self.volume_currency = self.currencies["volume"]
+        self.currency = self.currencies["base"]
+        name_suffix = self.currencies["volume"] + "-" + self.currencies["base"]
+        self.name = "BINANCEJE_" + name_suffix
+        self.friendly_name = "Binance Jersey " + name_suffix
+
+    def load_credentials(self):
+        credentials = ["api_key", "secret"]
+        if self.credentials is None:
+            self.credentials = {
+                credential: self._load_env("BINANCEJE_" + credential.upper())
+                for credential in credentials
+            }
+
+    def auth_request(self, req_method, url, request_args):
+        """Modify a request to add authentication header and query string.
+
+        Overrides the not implemented method from the base class.
+
+        For authenticated endpoints, Binance requires the api key in the
+        request header and a timestamp and signature within the query string.
+        """
+        self.load_credentials()
+
+        try:
+            headers = request_args["headers"]
+        except KeyError:
+            headers = request_args["headers"] = {}
+
+        headers["X-MBX-APIKEY"] = self.credentials["api_key"]
+
+        try:
+            params = request_args["params"]
+        except KeyError:
+            params = request_args["params"] = {}
+
+        timestamp = int(round(time.time() * 1000))
+        params["timestamp"] = timestamp
+
+        signature = binance_signature(params, self.credentials["secret"])
+        params["signature"] = signature
+
+        logger.debug("headers: %s" % headers)
+        logger.debug("request args: %s" % request_args)
+
+    def ping(self):
+        req = self.req(no_auth=True, **self.endpoints["ping"])
+        return self.resp(req)
+
+    def get_balance_req(self):
+        return self.req(**self.endpoints["balance"])
+
+    def get_balance_resp(self, req):
+        response = self.resp(req)
+        logger.debug("balance repsonse: %s" % response)
+
+        try:
+            balances = {
+                currency: Money(balance["free"], currency)
+                for balance in response["balances"]
+                for currency in Money.CURRENCIES
+                if balance["asset"] == currency
+            }
+            logger.debug("balances: %s" % balances)
+        except KeyError:
+            raise exceptions.ExchangeAPIErrorException(
+                self, "Cannot determine balances from response"
+            )
+
+        return Balance(balances)
+
+    def get_prices(self):
+        req = self.get_prices_req()
+        return self.get_prices_resp(req)
+
+    def get_prices_req(self):
```

Comment:
There is a function in the api spec for 'get_ticker'. Could we implement that instead of the prices endpoint to be consistent with the other wrappers? e.g. https://github.com/garethdmm/gryphon/blob/dbdcdc90c3cb982e28405c6485af4a9393512dce/gryphon/lib/exchange/bitstamp_btc_usd.py#L100-L111

Reply:
Will do.